routesrv/options.go

```diff
@@ -166,4 +166,7 @@ type Options struct {
 	// EditRoute will be applied to all routes automatically and
 	// will apply changes to all matching routes.
 	EditRoute []*eskip.Editor
+
+	// DisabledFilters is a list of filters unavailable for use
```

Comment:
What will happen if I apply a disabled filter if it's set here?
Do we ignore the filter or do we log an error and break update of the route?

These would be great to answer here.